gui/src/system/SystemPages.tsx

```diff
@@ -345,7 +357,7 @@ async function submitConfig(edited: boolean) {
       await new Promise(r => setTimeout(r, 1000));
```

Comment:
Is this actually the best way to do this? Maybe it's just JS but this seems so clunky.